src/dmd/expressionsem.d

```diff
@@ -604,7 +604,7 @@ private bool functionParameters(Loc loc, Scope* sc, TypeFunction tf, Type tthis,
                 nargs++;
             }
 
-            if (tf.varargs == 2 && i + 1 == nparams)
+            if (tf.varargs == 2 && i + 1 == nparams) // https://dlang.org/spec/function.html#variadic
```

Comment:
Maybe the link should be put on the line before this condition. Perhaps with a slight introduction also, eg: `Handle variadic arguments as per spec <link>`.

Reply:
The spec link implies just that.

Reply:
The comment still - to me at least - looks to be in an awkward place.  But well, eye of the beholder and all.